x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/MvExpandOperatorTests.java

```diff
@@ -110,34 +208,53 @@ protected ByteSizeValue smallEnoughToCircuitBreak() {
     }
 
     public void testNoopStatus() {
-        MvExpandOperator op = new MvExpandOperator(0);
+        MvExpandOperator op = new MvExpandOperator(0, randomIntBetween(1, 1000));
         List<Page> result = drive(
             op,
             List.of(new Page(IntVector.newVectorBuilder(2).appendInt(1).appendInt(2).build().asBlock())).iterator(),
             driverContext()
         );
         assertThat(result, hasSize(1));
         assertThat(valuesAtPositions(result.get(0).getBlock(0), 0, 2), equalTo(List.of(List.of(1), List.of(2))));
-        MvExpandOperator.Status status = (MvExpandOperator.Status) op.status();
-        assertThat(status.pagesProcessed(), equalTo(1));
+        MvExpandOperator.Status status = op.status();
+        assertThat(status.pagesIn(), equalTo(1));
+        assertThat(status.pagesOut(), equalTo(1));
         assertThat(status.noops(), equalTo(1));
     }
 
     public void testExpandStatus() {
-        MvExpandOperator op = new MvExpandOperator(0);
+        MvExpandOperator op = new MvExpandOperator(0, randomIntBetween(1, 1));
         var builder = IntBlock.newBlockBuilder(2).beginPositionEntry().appendInt(1).appendInt(2).endPositionEntry();
         List<Page> result = drive(op, List.of(new Page(builder.build())).iterator(), driverContext());
         assertThat(result, hasSize(1));
         assertThat(valuesAtPositions(result.get(0).getBlock(0), 0, 2), equalTo(List.of(List.of(1), List.of(2))));
-        MvExpandOperator.Status status = (MvExpandOperator.Status) op.status();
-        assertThat(status.pagesProcessed(), equalTo(1));
+        MvExpandOperator.Status status = op.status();
+        assertThat(status.pagesIn(), equalTo(1));
+        assertThat(status.pagesOut(), equalTo(1));
         assertThat(status.noops(), equalTo(0));
     }
 
-    // TODO: remove this once possible
-    // https://github.com/elastic/elasticsearch/issues/99826
-    @Override
-    protected boolean canLeak() {
-        return true;
+    public void testExpandWithBytesRefs() {
```

Comment:
@nik9000 I stole this from https://github.com/elastic/elasticsearch/pull/100548

Reply:
Nice.